src/features/components/templates/navigation/payment-agent-nav/index.tsx

```diff
@@ -35,6 +37,17 @@ const PaymentAgentAffiliateNav = ({ is_prime_page = false }: { is_prime_page?: b
               sign_up: '_t_Affiliate & IB sign up_t_',
           }
 
+    const partners_signup_ab_test = Analytics?.getFeatureValue(
+        'partners_signup_ab_test',
+        'fallback',
+    )
+    let language = getLanguage()
+    language = language !== 'en' ? '/' + language : ''
+    const affiliate_signup_link =
+        partners_signup_ab_test === true
+            ? window.location.origin + language + '/signup-affiliates'
+            : affiliate_signup_url
+
```

Comment:
1. No memo?
2. Are you interested in explicit `true`?
3. Why not inline these calculations?

Reply:
I'm working on creating hook for these needs, thanks for comment :)